garden-backend-service/src/config.py

```diff
@@ -58,7 +58,7 @@ class Settings(BaseSettings):
     def SQLALCHEMY_DATABASE_URL(self) -> str:
         return f"postgresql+asyncpg://{self.DB_USERNAME}:{self.DB_PASSWORD}@{self.DB_ENDPOINT}/garden_db_{self.GARDEN_ENV}"
 
-    model_config = SettingsConfigDict(env_file=_dotenv_path, case_sensitive=True)
+    model_config = SettingsConfigDict(env_file=_dotenv_path, case_sensitive=True, extra='ignore')
```

Comment:
I think this is sane either way but I'd just as soon keep the default here and use a different env file (`.postgres.env`, maybe?) for the postgres variables.

Thinking along the lines of keeping the local "deployment" as close to the real deployment as possible, we should reserve the existing `.env` just for providing/overriding environment variables the app actually knows about

Reply:
That makes sense, a separate .env for postgres is no problem.